bin/build-drivers/deps.edn

```diff
@@ -1,40 +0,0 @@
-{:paths ["src"]
-
- :deps
- {org.clojure/clojure             {:mvn/version "1.11.1"} ; explicit otherwise version is picked up from cli
-  common/common                   {:local/root "../common"}
-  com.github.seancorfield/depstar {:mvn/version "2.1.278"}
-  cheshire/cheshire               {:mvn/version "5.8.1"}
-  commons-codec/commons-codec     {:mvn/version "1.14"}
-  expound/expound                 {:mvn/version "0.7.0"} ; better output of spec validation errors
-  hiccup/hiccup                   {:mvn/version "1.0.5"}
-  io.forward/yaml                 {:mvn/version "1.0.9"} ; Don't upgrade yet, new version doesn't support Java 8 (see https://github.com/owainlewis/yaml/issues/37)
-  io.github.clojure/tools.build   {:git/tag "v0.7.4" :git/sha "ac442da"}
-  org.clojure/tools.deps.alpha    {:mvn/version "0.12.985"}
-  com.bhauman/spell-spec          {:mvn/version "0.1.1"} ; used to find misspellings in YAML files
-  stencil/stencil                 {:mvn/version "0.5.0"}
-  ;; local source
-  metabase/metabase-core          {:local/root "../.."}
-  metabase/driver-modules         {:local/root "../../modules/drivers"}}
-
- ;; These are needed for the Athena and Redshift drivers in order to build them. Maven repos from subprojects do not
- ;; get copied over -- see
- ;; https://ask.clojure.org/index.php/10726/deps-manifest-dependencies-respect-repos-dependent-project
- :mvn/repos
- {"athena"   {:url "https://s3.amazonaws.com/maven-athena"}
-  "redshift" {:url "https://s3.amazonaws.com/redshift-maven-repository/release"}
-  ;; for metabase/saml20-clj
-  "opensaml" {:url "https://build.shibboleth.net/nexus/content/repositories/releases/"}}
-
- :jvm-opts
- ["-XX:-OmitStackTraceInFastThrow"]
-
- :aliases
- {:dev
-  {:extra-paths ["test"]}
-
-  :test
-  {:extra-paths ["test"]
-   :extra-deps  {com.cognitect/test-runner {:git/url "https://github.com/cognitect-labs/test-runner.git"
-                                            :sha     "209b64504cb3bd3b99ecfec7937b358a879f55c1"}}
-   :main-opts   ["-m" "cognitect.test-runner"]}}}
```

Comment:
We don't need different test runners for the build scripts vs everything else now